ground-server/src/lib/telemetry-channels.ts

```diff
@@ -115,4 +115,117 @@ export const TELEMETRY_CHANNELS: TelemetryChannel[] = [
     modes: ["Value"],
     component: GraphWidget,
   },
+  // TODO: Add metadata fields in telemetry channels 
+  {
+    id: "alt_armed",
+    label: "Altimeter Armed",
+    requiresAuth: false,
+    dbField: "alt_armed",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "alt_valid",
+    label: "Altimeter Valid",
+    requiresAuth: false,
+    dbField: "alt_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "gps_valid",
+    label: "GPS Valid",
+    requiresAuth: false,
+    dbField: "gps_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "imu_valid",
+    label: "IMU Valid",
+    requiresAuth: false,
+    dbField: "imu_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "acc_valid",
+    label: "Acceleromter Valid",
+    requiresAuth: false,
+    dbField: "acc_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "therm_valid",
+    label: "Thermometer Valid",
+    requiresAuth: false,
+    dbField: "therm_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "voltage_valid",
+    label: "Voltage Valid",
+    requiresAuth: false,
+    dbField: "voltage_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "adc_valid",
+    label: "ADC Valid",
+    requiresAuth: false,
+    dbField: "adc_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "fram_valid",
+    label: "FRAM Valid",
+    requiresAuth: false,
+    dbField: "fram_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "sd_valid",
+    label: "SD Valid",
+    requiresAuth: false,
+    dbField: "sd_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "gps_msg_valid",
+    label: "GPS Message Valid",
+    requiresAuth: false,
+    dbField: "gps_msg_valid",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "mav_state",
+    label: "MAV State",
+    requiresAuth: false,
+    dbField: "mav_state",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "sv_state",
+    label: "SV State",
+    requiresAuth: false,
+    dbField: "sv_state",
+    modes: ["Value"],
+    component: GraphWidget,
+  },
+  {
+    id: "flight_mode",
```

Comment:
If you're storing this as a string, it won't play nice with the graphwidget

Reply:
Good point